mslice/tests/cut_presenter_test.py

```diff
@@ -159,13 +159,8 @@ def test_cut_parse_input_errors(self):
         cut_presenter.notify(Command.Plot)
         self.assertRaises(ValueError)
         # Bad cut axis
-        axis = Axis("units", "a", "100", "1")
-        cut_presenter = CutPresenter(self.view, self.cut_algorithm, self.cut_plotter)
-        cut_presenter.register_master(self.main_presenter)
-        self._create_cut(axis, processed_axis, integration_start, integration_end, width,
-                         intensity_start, intensity_end, is_norm, workspace, integrated_axis)
-        cut_presenter.notify(Command.Plot)
-        self.assertRaises(ValueError)
+        with self.assertRaises(ValueError):
```

Comment:
I didn't know you could use the assert statements as context managers - this is far better that the standard way of using this. :+1: